pyannote/audio/features/precomputed.py

```diff
@@ -91,7 +91,7 @@ def __init__(self, root_dir=None, use_memmap=True,
         if path.exists():
 
             with io.open(path, 'r') as f:
-                params = yaml.load(f)
+                params = yaml.load(f, Loader=yaml.Loader)
```

Comment:
Sometimes you use `yaml.Loader`, sometimes `yaml.SafeLoader`.
Which one should be used?

Reply:
That's a mistake on my side, safe loader is the the only loader